### ISSUE TITLE
Style Footer Component

### ISSUE THREAD

**Issue by ezioda004** [UdacityFrontEndScholarship/quiz-app#40]
## Style the Footer Component as shown in this picture.
![image](https://user-images.githubusercontent.com/20020273/38775935-f9971c9a-40ab-11e8-9399-549a88bc98d2.png)

Make sure the Footer is responsive as well.

How to get started? Read our [project setup instructions](https://github.com/UdacityFrontEndScholarship/quiz-app/blob/master/CONTRIBUTING.md#project-setup-instructions).
Fork the project, navigate to directory quiz-app => src => Component => Footer.css and Footer.js

### All you need to do is write normal CSS inside Footer.css.
We are using Bootstrap-4 for this project but feel free to use native CSS wherever required.

Note:

1. If you use any self closing tags such as `<img>`, `<input>`, `<br>`
2. Make sure you always close them like this `<img />`, `<input />`, `<br />`

If you add a class to an element example
```
<div  class = "nav navbar"> 
    <!-- rest of your code -->
</div>
```
Make sure you use className instead of class. Example
```
<div  className = "nav navbar"> 
    <!-- rest of your code -->
</div>
````

This is fairly easy and straightforward to do, please comment on this issue and I'll assign it to you.

**Comment by jibin2706** [comment]
Assign this to me

**Comment by SukhjinderArora** [comment]
Can I work on it ?

**Comment by ezioda004** [comment]
Assigning to @jibin2706, feel free to contact us if you get stuck anywhere.
@SukhjinderArora We'll make more issues, please wait till we do so. 😇

**Comment by jibin2706** [comment]
@ezioda004 Is that a border around footer?

**Comment by ezioda004** [comment]
@jibin2706 The box is just for mockup purpose, to bundle all the links, visualizing which font properties to be used.

**Comment by jibin2706** [comment]
#515B69 is the background color or font color?

**Comment by ezioda004** [comment]
@jibin2706 Thats background-color with 8% opacity. Alternatively you can use `#F1F2F3` with 100% opacity instead.

**Comment by ezioda004** [comment]
Resolved by PR #43 
Closing this issue.
